libevolve/GA_API.py

```diff
@@ -0,0 +1,113 @@
+import itertools
+import string
+import numpy as np
+from libevolve.libevolve.ga.SGA  import SGA
+from flask import Flask,request,jsonify
+import logging as logger
+
+logger.basicConfig(level="DEBUG")
+
+
+
+
+
+
+
+
+def objective_fn(param,w):
+    """ Sample objective function for evolutionary algorithm
```

Comment:
Use `numpy` docstring instead of the `rStructredText` for consistency.

Reply:
i copied it from the code you previously sent to us

Reply:
Ah. Anyway, try to change your editor to `numpy` docstring instead of the `rStructredText` for the future code snippets.